### ISSUE TITLE
ffe-datepicker-react: Setting an ID is unintuitive

### ISSUE THREAD

**Issue by wkillerud** [SpareBank1/designsystem#189]
#### Which package is this issue related to?

`ffe-datepicker-react`

#### Describe your issue

`Datepicker` lets us provide a `label` prop to get a nice form label related to the input. What isn't immediately obvious is that `label` needs `inputProps.id` in order to work. So this will fail a11y validation: 

```
<Datepicker
    label="Velg dato"
    language="nb"
    maxDate="31.12.2016"
    minDate="01.01.2016"
    onChange={date => setState({ date })}
    value={state.date}
/>
```

And this is required to pass:

```diff
    <Datepicker
+       inputProps={{ id: 'datepicker-example' }}
        label="Velg dato"
        language="nb"
        maxDate="31.12.2016"
        minDate="01.01.2016"
        onChange={date => setState({ date })}
        value={state.date}
    />;
```

#### What is the expected behavior? 

I'd like it if the root level `id` prop was passed on to the input field.

```diff
    <Datepicker
+       id="datepicker-example"
        label="Velg dato"
        language="nb"
        maxDate="31.12.2016"
        minDate="01.01.2016"
        onChange={date => setState({ date })}
        value={state.date}
    />;
```

It would also be nice if the component, like some of our other components, generates an ID for us if it's not given one.

#### What is the actual behavior?

It's easy to mess up a11y :crying_cat_face:

**Comment by selbekk** [comment]
I skipped the last part, but a pr is available at #267

**Comment by selbekk** [comment]
Closed the PR due to a disagreement over the very premise of this issue (having two ways to set the ID is worse than it being easy to set). Closing this issue, but I'm open for re-opening it later if need be.
